packages/ferreiro-client/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@ferreiro/client",
+  "version": "0.0.0",
+  "private": false,
+  "dependencies": {
+    "@ferreiro/design-system": "/Users/ferreiro/Documents/website/packages/design-system",
```

Comment:
oh! Good point. I also don't like to include the local path. I played with `yarn link` too and it's super cool to be able to use a local copy of an external dependecy. 

That's said, since the `design_system` package is not yet published, what should I put in the value for `@ferreiro/design-system:`? I was confused about it.

Reply:
Use the value from its `package.json`, yarn will handle the rest.